lib/client.js

```diff
@@ -859,6 +863,21 @@ class Parser {
       socket[kReset] = true
     }
 
+    for (let i = 0; i < headers.length; i+= 2) {
+      if (headers[i].toString().toLowerCase() === 'keep-alive') {
+        const value = headers[i + 1].toString().toLowerCase();
+        const maxIndex = value.indexOf('max=');
+
+        if (maxIndex !== -1) {
+          const pipelining = parseInt(value.slice(maxIndex + 4));
+          if (Number.isFinite(pipelining)) {
+            // What if client[kPipelining] is undefined?
```

Comment:
?

Reply:
Should we make `client[kPipelining]` to always be a number?

Reply:
I think so

Reply:
@ronag wdyt?